ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -18,21 +18,23 @@ KUBECONFIG="" oc --loglevel=8 registry login
 
 # Print cv, failed node, co, mcp information for debug purpose
 function debug() {
-    if (( FRC != 0 )); then
-        echo -e "oc get clusterversion/version -oyaml\n$(oc get clusterversion/version -oyaml)"
-        echo -e "oc get machineconfig\n$(oc get machineconfig)"
-        echo -e "Describing abnormal nodes...\n"
+    # to delete - temporary enable debug every time
+    if true ; then
+        echo -e "\n# oc adm upgrade status\n$(env OC_ENABLE_CMD_UPGRADE_STATUS='true' oc adm upgrade status)"
```

Comment:
The `cucushift-upgrade-toimage` is being used in many upgrade jobs, will the command bring some side effect on the old version of `oc` ?

Reply:
I have tested locally, `oc adm upgrade status` acts as `oc adm upgrade` in older oc clients that do not support the command.
in function dump_status_if_unexpected() that will run multiple times per upgrade, i have implemented a filter to run the command only on 4.16+ so we won't pollute the logs, but here in func debug I'm not sure if we really need to filter :thinking: the only side effect if we don't, is `oc adm upgrade` output once in the log for older clusters
WDYT?

Reply:
okay, if the `status` option on unsupported oc version does not break anything, then it can be accepted.